.eslintrc

```diff
@@ -4,6 +4,13 @@
   "env": {
     "browser": true
   },
+  "globals": {
+    "globalThis": true
```

Comment:
did this literally last night in a side project and didn't have to change my .eslintrc (https://github.com/Dashron/Roads/commit/c4005e4db99caa5234760581d396b7520f4b6377#diff-d588feea505a1ca70e4e3e31304ba1e59728683926ebdda69e33e8972939627fR10-R19). maybe it's because the "env" is just browser? My code has browser node and es2021.

Reply:
Was able to drop this global by changing the `browser: true` env to `"es2020": true`.